src/OpenAIP.cpp

```diff
@@ -81,6 +82,7 @@ bool OpenAIP::Read(const std::string& fileName, std::multimap<int, Airspace>& ou
 	read_xml(input, tree);
 	input.close();
 	ptree root;
+	setlocale(LC_ALL, "C"); // so std::stod recognize dot as decimal sep
```

Comment:
Questo risolve il problema, ma forse sarebbe meglio farlo nel main.cpp dell'applicazione Qt, da testare se cosi funziona.

Reply:
questa sera ci provo, lo metto in linea 17 di main.cpp dell'applicazione Qt